lib/galaxy/dependencies/__init__.py

```diff
@@ -203,7 +203,7 @@ def check_total_perspective_vortex(self):
     def check_pbs_python(self):
         return "galaxy.jobs.runners.pbs:PBSJobRunner" in self.job_runners
 
-    def check_pykube(self):
+    def check_pykube_ng(self):
```

Comment:
I don't think we want to merge https://github.com/galaxyproject/galaxy/pull/17323 forward, since it was the original PR was merged again in dev via https://github.com/galaxyproject/galaxy/pull/17324 .